packages/babel-helpers/src/helpers.js

```diff
@@ -399,23 +399,31 @@ helpers.inherits = () => template.program.ast`
     if (typeof superClass !== "function" && superClass !== null) {
       throw new TypeError("Super expression must either be null or a function");
     }
-    setPrototypeOf(subClass.prototype, superClass && superClass.prototype);
+    subClass.prototype = Object.create(superClass && superClass.prototype, {
+      constructor: {
+        value: subClass,
+        enumerable: false,
```

Comment:
It could be better to omit `enumerable: false` just to not force sham implementations to throw, but this would only happen on IE<9 anyway.

Reply:
This is the spec inherits. I think changing it to be enumerable would break more people who use a for-in.

You can always use loose mode to support these IE9?

Reply:
I think they just mean that the default is already `false` anyway, so leaving it out doesn't make anything fail, but allows polyfills to be less agressive in requiring enumerability on older platforms.

Reply:
Ahh, ok.